peer/brontide.go

```diff
@@ -4111,6 +4118,29 @@ func (p *Brontide) handleRemovePendingChannel(req *newChannelMsg) {
 	p.addedChannels.Delete(chanID)
 }
 
+// handlePeerStorageMessage handles `PeerStorage` message, it stores the message
+// and sends it back to the peer as an ack.
+func (p *Brontide) handlePeerStorageMessage(msg *lnwire.PeerStorage) error {
+	// Check if we have the feature to store peer backup enabled.
+	if p.cfg.PeerDataStore != nil {
+		// Store the data.
+		err := p.cfg.PeerDataStore.Store(msg.Blob)
+		if err != nil {
+			return err
+		}
+
+		// Send back the data as an ack.
+		err = p.SendMessage(false, lnwire.NewYourPeerStorageMsg(
+			msg.Blob),
+		)
```

Comment:
looks like this part of the spec might change to just be an ACK along with an ID

Reply:
Updated Spec still has the same behavior with no ID